ProtectionScan/Features/MainFeature.cs

```diff
@@ -55,6 +63,9 @@ public MainFeature()
 
             Add(DebugInput);
             Add(FileOnlyInput);
+            #if NETCOREAPP
```

Comment:
Non-obvious formatting nit: `#if`, `#elif`, and `#else` preprocessor directives should be aligned fully to the left.